libdnf/CMakeLists.txt

```diff
@@ -61,7 +63,8 @@ TARGET_LINK_LIBRARIES(libdnf
                       ${GLIB_GIO_UNIX_LIBRARIES}
                       ${LIBSOLV_LIBRARY}
                       ${LIBSOLV_EXT_LIBRARY}
-                      ${RPMDB_LIBRARY})
+                      ${RPMDB_LIBRARY}
+                      /usr/lib64/libmodulemd.so)
```

Comment:
@ignatenkobrain Hi, can you take a look at this? ${LIBMODULEMD_LIBRARY} is not working for me.

Reply:
it should be `${LIBMODULEMD_LIBRARIES}`